src/pretix/base/views/mixins.py

```diff
@@ -96,7 +96,7 @@ def forms(self):
                 ))
             )
 
-            override_sets = self.get_question_override_sets(cr)
+            override_sets = self.get_question_override_sets(cr, idx)
```

Comment:
Based on idx, there is one edge-case where this does not work: when I have a non-admission product as the first product in my basket, it does not auto-fill the first admission-product with the name. Probably very very rare and so rare enough to be ignored?

Reply:
Yeah, I think that's fine.